### PR TITLE
show cart in header

### PR DESCRIPTION
**Pull request by ghost**
## Description
show cart in header
## Reference Requirements
- Task ID: TI-1

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

### CODE REVIEW

File: resources/views/user/layout/header.blade.php [hunk 1]
```diff
@@ -51,16 +51,28 @@
...
                 </a>
-              <!-- </div> -->
+              </div>
+              <div>
+                <div class="top-cart-content">
+                  <div class="block-subtitle hidden-xs">Recently added item(s)</div>
+                    <ul id="cart-sidebar" class="mini-products-list">
+                      
+                    </ul>
+                  <div class="top-subtotal">Subtotal: <span class="price">$<span class="sub-total">0</span></span></div>
```
Comment (at-sonle) [inline resources/views/user/layout/header.blade.php]: dung &#36 và lang để xuất subtotal

File: resources/views/user/layout/header.blade.php [hunk 2]
```diff
@@ -51,16 +51,28 @@
...
+                  <div class="actions">
+                    <a href="{{ route('cart.index') }}" class="view-cart"><i class="fa fa-shopping-cart"></i> <span>View Cart</span></a>
```
Comment (at-sonle) [inline resources/views/user/layout/header.blade.php]: mấy cái này dùng lang xuât ra

File: resources/views/user/layout/header.blade.php [hunk 3]
```diff
@@ -51,16 +51,28 @@
...
+                  <span class="cart-title">{{ __('home.user.header.cart') }}</span>
+                  <span id="number-item">0</span>
+                  <span class="cart-total">{{ __('home.user.header.items') }}</span>
+                </div>
                 </a>
-              <!-- </div> -->
+              </div>
+              <div>
+                <div class="top-cart-content">
+                  <div class="block-subtitle hidden-xs">Recently added item(s)</div>
```
Comment (at-sonle) [inline resources/views/user/layout/header.blade.php]: dùng lang ra

**Comment by at-sonle** [comment]
LGTM

**Review by mvn-quangtran-dn — APPROVED** [review]
LGTM

**Pull request merged**